test/e2e/util.ts

```diff
@@ -85,6 +87,20 @@ export class E2eHelper {
     return workspaceStatus
   }
 
+  async waitWorkspaceStatus(status: string, timeoutMs: number): Promise<boolean> {
```

Comment:
Usually `wait` methods return void or throw an error.

Reply:
The idea is to wait for an event and then decide what to do: fail or continue. But maybe it is not needed for tests...